### PR TITLE
Misc. formatting fixes

### PR DESCRIPTION
**Pull request by cl8n**
don't review this yet, I'll be redoing commits as necessary

- [x] HitObject -> Hit object (changing caps where necessary)
- [x] Remove whitespace from tables
- [ ] Standardize image filenames (only hyphens for separating words) + ASC
- [x] Remove code styling from headers
- [ ] Match article titles to their folder names (except cases where it makes sense to have different) - waiting on #1849 
- [ ] Link users/beatmaps/etc by ID instead of name
- [x] Remove tabs
- [x] Remove trailing (single) spaces
- [ ] Remove other kinds of whitespace not in ASCII

---

### CODE REVIEW

File: news/2013-10-08-storyboard-showcase-kozato-45nen-no-yukizakura.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Without further ado, I present to you the first showcased map:
 **[Kozato - 45nen no Yukizakura](https://osu.ppy.sh/s/50771) by [Rewrite](https://osu.ppy.sh/u/1161361)**  
 **Storyboard by [yukinohara](https://osu.ppy.sh/u/580107)**
 
-This map was lucky enough to have both a custom skin and storyboard created for it by yukinohara, a very skilled graphic designer. The pink, cherry blossom-themed images subtly move and transition to set a wonderful atmosphere for the music, and the very sleek custom skin ensures that this atmosphere is accented by the hitobjects, rather than harmed.
+This map was lucky enough to have both a custom skin and storyboard created for it by yukinohara, a very skilled graphic designer. The pink, cherry blossom-themed images subtly move and transition to set a wonderful atmosphere for the music, and the very sleek custom skin ensures that this atmosphere is accented by the hit objects, rather than harmed.
```
Comment (MegaApplePi) [inline news/2013-10-08-storyboard-showcase-kozato-45nen-no-yukizakura.md]: Changing it in the news too?

Comment (cl8n) [reply news/2013-10-08-storyboard-showcase-kozato-45nen-no-yukizakura.md]: yeah, it's not a matter of writing style and it doesn't change the meaning or anything so I went for it

Comment (MegaApplePi) [reply news/2013-10-08-storyboard-showcase-kozato-45nen-no-yukizakura.md]: alright, sounds good

**Comment by cl8n** [comment]
e8b5e68120246795bfc9da1b472d99069bd2b4c3 has a lot of changes so don't try to view that one in a browser

it's the result of the following command + fixing up cases where pipes were used outside of tables
```sh
find . -name '*.md' -exec sed -i \
    -e 's/|\(\| \{2,\}\)\([^| ][^|]*\) *|/| \2 |/g' \
    -e 's/| *\([^|]*[^| ]\)\(\| \{2,\}\)|/| \1 |/g' \
    -e 's/| \([:-]\)-\{2,\}\([:-]\) |/| \1-\2 |/g' \
    {} +
```

**Comment by cl8n** [comment]
same with 26644d8970620d725a5932cc224a24d7ffb67761
```sh
find . -name '*.md' -exec sed -i -e 's/\([^ ]\) $/\1/g' {} +
```

**Comment by cl8n** [comment]
I will split all of these into their own PR where possible, if I keep them together it's going to take so long to merge I'll be fixing conflicts over and over

**Pull request closed without merging**